CommunityTracker/CommunityTracker.API/Controllers/CommunityController.cs

```diff
@@ -60,41 +61,61 @@ public async Task<IActionResult> GetAllManagers()
         /// <summary>
         /// Adds the community.
         /// </summary>
-        /// <param name="request">The API dto.</param>
+        /// <param name="apiDTO">The API dto.</param>
         /// <returns></returns>
         [HttpPost]
-        public async Task<IActionResult> AddCommunity([FromBody] AddRequestDTO request)
+        public async Task<IActionResult> AddCommunity([FromBody] AddRequestDTO apiDTO)
         {
-            if (request == null)
+            var communityDTO = new CommunityDTO();
+            communityDTO.CommunityName = apiDTO.CommunityName;
+            communityDTO.CommunityMgrid = apiDTO.CommunityManager;
+            communityDTO.CommunityDesc = apiDTO.Description;
+            var result = await _communityServiceCommands.AddCommunity(communityDTO);
+            if (result is null)
             {
-                return null;
+                return BadRequest(new CustomErrors()
+                {
+                    result = new Result()
+                });
             }
-
-            var communityDTO = new CommunityDTO
+            var response = new ResponseDTO()
             {
-                CommunityName = request.CommunityName,
-                CommunityMgrid = request.CommunityManager,
-                CommunityDesc = request.Description
+                CommunityId = result.CommunityId,
+                CommunityName = result.CommunityName,
+                CommunityManager = result.CommunityManagerName,
+                Description = result.CommunityDesc
             };
+            return Ok(response);
+        }
```

Comment:
```suggestion
        }
```
Revert to previous code, just add isActive field in responseDTO